### DISCUSSION TITLE
Circle goes round and round in the new version and never stops.

### DISCUSSION THREAD

**Discussion by Philrobots** [tomik23/circular-progress-bar#86] [Announcements]
**Video :** 

https://github.com/tomickigrzegorz/circular-progress-bar/assets/47373969/e6661cb0-262e-4e75-a2d4-0e0d427aecc4

**What happens :** 

So I installed the new version in my application to fix the circle error, which remains at 0.

But now, from time to time, the circle goes round and round in circles, even if I give it a percentage of 95%.

What could be causing this?

**Version :** 

Latest (v1.2.0)

**Comment by tomickigrzegorz** [comment]
I asked if you had tested it :)
Prepare an example on codepen.io because we won't be able to agree on this.
I prepared https://codepen.io/Tomik23/pen/QWYRWJd and everything works fine for me.

**Comment by Philrobots** [comment]
Interesting, it indeeds work fine. 

Let me take a deeper look
